lib/productive.rb

```diff
@@ -12,6 +12,9 @@
 require 'productive/resources/invoice'
 require 'productive/resources/person'
 require 'productive/resources/project'
+require 'productive/resources/service_type'
```

Comment:
I see you've reordered the files, but I think this is due to the has_one association that's using the ServiceType class instead of symbol.
The alphabetical list should be ok once you've sorted that out?